### PR TITLE
added autolinking in appintro_fragment_intro.xml

### PR DESCRIPTION
**Pull request by MichaelPlug**
I added in the xml code a new line for use the auto-linking. In this way, is possible to tap a URL to open it in the browser.

### CODE REVIEW

File: appintro/src/main/res/layout-land/appintro_fragment_intro.xml [hunk 1]
```diff
@@ -13,7 +13,6 @@
         android:id="@+id/title"
         style="@style/AppIntroDefaultHeading"
         android:layout_width="0dp"
-        android:layout_height="wrap_content"
```
Comment (cortinico) [inline appintro/src/main/res/layout-land/appintro_fragment_intro.xml:16]: Why have you removed this?

File: appintro/src/main/res/layout/appintro_fragment_intro.xml [hunk 2]
```diff
@@ -40,6 +40,7 @@
         style="@style/AppIntroDefaultText"
         android:layout_width="0dp"
         android:layout_height="0dp"
+        android:autoLink="all"
```
Comment (cortinico) [inline appintro/src/main/res/layout/appintro_fragment_intro.xml:43]:
```suggestion
        android:autoLink="web"

```

File: appintro/src/main/res/layout-land/appintro_fragment_intro.xml [hunk 3]
```diff
@@ -37,6 +36,7 @@
         style="@style/AppIntroDefaultText"
         android:layout_width="0dp"
         android:layout_height="wrap_content"
+        android:autoLink="all"
```
Comment (cortinico) [inline appintro/src/main/res/layout-land/appintro_fragment_intro.xml:39]:
```suggestion
        android:autoLink="web"
```

**Review by cortinico — CHANGES_REQUESTED** [review]
Thanks for the contribution @MichaelPlug

File: appintro/src/main/res/layout-land/appintro_fragment_intro.xml (hunk 1 above)
Comment (MichaelPlug) [reply appintro/src/main/res/layout-land/appintro_fragment_intro.xml:16]: Sorry, I noticed this immediately after the push.

Comment (MichaelPlug) [reply appintro/src/main/res/layout-land/appintro_fragment_intro.xml:16]: Sorry, I noticed this change only after the push. I was trying to undo it.

Comment (MichaelPlug) [reply appintro/src/main/res/layout-land/appintro_fragment_intro.xml]: I fixed it

**Review by cortinico — APPROVED** [review]

**Pull request merged**